pkg/fw/Dockerfile

```diff
@@ -61,6 +61,12 @@ ADD ${RPI_BT_FIRMWARE_URL}/${RPI_BT_FIRMWARE_VERSION}/broadcom/BCM4345C0.hcd .
 ADD ${RPI_BT_FIRMWARE_URL}/${RPI_BT_FIRMWARE_VERSION}/broadcom/BCM43430B0.hcd .
 ADD ${RPI_BT_FIRMWARE_URL}/${RPI_BT_FIRMWARE_VERSION}/broadcom/BCM4345C5.hcd .
 
+# Hailo GPU firmware
+# we've got a written permission to include this firmware into EVE
```

Comment:
Well, I think we must take this with precaution. Although it's a firmware blob, so we are not talking about source code to be included on EVE, this written permission should be compatible with Apache 2.0, which means that it should not forbid us in anyway from any of the permissions guaranteed by our license: rights to use the software as it is, rights to change it (EVE), rights to share it... since it's a firmware blob, perhaps the most important is the right to share it as we wish (within EVE, of course)... I think it's important to include a license file here, we do have proprietary firmware in this package (i.MX, NVIDIA), but they have specific (proprietary) licenses that allows to share, pack with EVE, etc, i.e., which doesn't break the terms of our license...

Reply:
@rene yes, sure. I've submitted this PR as a placeholder so I do not loose the code. still waiting for confirmation and proper license file.

Reply:
It might be feasible to download it during VM deployment.

Reply:
> It might be feasible to download it during VM deployment.
not really, the end user must accept EULA but we managed to get approval from Hailo